content/articles/2021-09-september-updates.md

```diff
@@ -0,0 +1,21 @@
+Title: August updates
```

Comment:
Wooooooooooooooooooooops 🙈 @jorgepiloto @Yash-10 haha we all missed this

Reply:
Fixed in https://github.com/poliastro/poliastro.github.io/commit/d0ffc71663b603fc6fcba8340f9fba935882a318